src/feature/feature.ts

```diff
@@ -24,8 +24,11 @@ const getFeature = <F extends FeatureType>(
     const url = await getFeatureURL(featureType, featureOptions, fullscriptOptions, frameId);
     mountPoint = document.getElementById(elementId);
     validateMountPoint(mountPoint);
-    const iframe = createIframe(url);
-    mountPoint.appendChild(iframe);
+
+    if (!mountPoint.querySelector(`#${WRAPPER_DIV_ID}`)) {
+      const iframe = createIframe(url);
+      mountPoint.appendChild(iframe);
```

Comment:
Won't this prevent re-mounting indefinitely? I vaguely recall hearing about some partners having a workflow where they intentionally do this after certain events like `patient.created`.

I think you need to make sure this only prevents a duplicate iframe that is using the same secret token. If they create a new feature, and use the same element ID to mount it, that should still work. 

Likely having a unique id to look for, and otherwise emptying the mount point if it's a new feature with an old iframe sitting in the mount point is how this needs to be handled.

Also I'm not sure you actually need a wrapper div. You might be able to just add the id to the iframe itself and base your logic on that.

Reply:
ohh interesting. That might explain what this particular partner might've been doing to cause the `remounting`. 

Okay, so what we need to do is:
1. Check if there's already an iframe (using the iframe id)
2. If there is, remove it first before mounting a new one

I'll try and test remounting based on an event to see if we can replicate an actual use-case 👍 

>If they create a new feature, and use the same element ID to mount it, that should still work.

Are you referring to a non-fullscriptjs feature here? It should still work with this implementation, yeah, because we're only targeting the div that we added